src/reconcile/rec_write.c

```diff
@@ -2341,6 +2355,7 @@ __wt_rec_cell_build_ovfl(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_KV *k
         dsk = tmp->mem;
         memset(dsk, 0, WT_PAGE_HEADER_SIZE);
         dsk->type = WT_PAGE_OVFL;
+        dsk->write_gen = ++btree->write_gen;
```

Comment:
Does write gen needed for overflow pages? I think we don't do any visibility check on overflown pages. They are just to store the large values. But I guess it is benign here to increase the btree write gen here.

I actually doubt that the overflow page is stored in the btree. I think it should just be a separate block.

Reply:
Yes. Overflow pages also have a write generation number, that gets updated when the page is written. The write generation number is not for visibility checks, it is to indicate the page numbers on the disk to identify which page image is greater compared to the other with a similar key range.